src/golang/lib/models/schema.go

```diff
@@ -0,0 +1,4 @@
+package models
+
+// SchemaVersion is the current database schema version.
```

Comment:
Drive-by-comment from your slack message :P Can you put a big capital note here that this is the source of truth? (with whatever caveats there are)

Reply:
Sure will add in a diff related PR I'm about to merge.